.github/workflows/ci.yml

```diff
@@ -82,28 +76,32 @@ jobs:
             temp/unity-packages/Library/ScriptAssemblies/*.TestRunner.*
           key: samples/unity-of-bugs|${{ steps.env.outputs.unityVersion }}-${{ hashFiles('samples/unity-of-bugs/Packages/packages-lock.json') }}
 
-      - uses: actions/download-artifact@v2
+      - name: Start the Unity the docker container
+        run: docker run -td --name unity -v ${{ github.workspace }}:/sentry-unity --workdir /sentry-unity ghcr.io/getsentry/unity-docker:editor-ubuntu-${{ steps.env.outputs.unityVersion }}-main
+
+      - uses: vaind/download-artifact@8bd9f2d7956f47bf3969323f3b2bc1b80987ef79
         with:
           name: Android-sdk
           path: package-dev/Plugins/Android
+          waitTimeout: 3600
 
-      - uses: actions/download-artifact@v2
+      - uses: vaind/download-artifact@8bd9f2d7956f47bf3969323f3b2bc1b80987ef79
         with:
           name: Android-libraries
           path: modules/sentry-java/sentry-android-ndk/build/intermediates/merged_native_libs/release/out/lib
+          waitTimeout: 3600
 
-      - uses: actions/download-artifact@v2
+      - uses: vaind/download-artifact@8bd9f2d7956f47bf3969323f3b2bc1b80987ef79
         with:
           name: iOS-sdk
           path: package-dev/Plugins/iOS
+          waitTimeout: 3600
 
-      - uses: actions/download-artifact@v2
+      - uses: vaind/download-artifact@8bd9f2d7956f47bf3969323f3b2bc1b80987ef79
```

Comment:
Do you plan to send these changes upstream to get back to using the official download action?
If not, we're likely going to need this fork under `getsentry` org

Reply:
yes, [I've already created one](https://github.com/actions/download-artifact/pull/139). As for if/when it's going to be merged... I can't tell

Reply:
On the other hand, how is using a fork in my repo and a specific git tag any different than doing that for other third-party actions, like this one? https://github.com/getsentry/sentry-unity/blob/701ca89b729b613ace2597498b1f6d420577ac54/.github/workflows/ci.yml#L27

Reply:
It's not. But given the fact this fork is being done specifically for this project, if we plan on maintaining, it's best it becomes part of the getsentry org as it's possible it'll serve this project longer than you and I